zone/questmgr.cpp

```diff
@@ -573,21 +549,17 @@ void QuestManager::settimer(const std::string& timer_name, uint32 seconds, Mob*
 
 	QTimerList.emplace_back(QuestTimer(seconds * 1000, mob, timer_name));
 
-	if (has_start_event) {
-		const std::string& export_string = fmt::format(
-			"{} {}",
-			timer_name,
-			seconds * 1000
-		);
-
-		if (mob->IsClient()) {
-			parse->EventPlayer(EVENT_TIMER_START, mob->CastToClient(), export_string, 0);
-		} else if (mob->IsBot()) {
-			parse->EventBot(EVENT_TIMER_START, mob->CastToBot(), nullptr, export_string, 0);
-		} else if (mob->IsNPC()) {
-			parse->EventNPC(EVENT_TIMER_START, mob->CastToNPC(), nullptr, export_string, 0);
+	parse->EventPlayerNPCBotMerc(
+		EVENT_TIMER_START,
+		mob,
+		[&]() {
+			return fmt::format(
+				"{} {}",
+				timer_name,
+				seconds * 1000
+			);
```

Comment:
It looks like this lambda is re-used without the arguments changing, so it could just be defined once before the first EVENT_TIMER_START and passed to both `parse->EventPlayerNPCBotMerc(EVENT_TIMER_START, mob, exportfn);`

Same with all the other timer events below